src/components/state-management/ServerState/server-state.mdx

```diff
@@ -0,0 +1,96 @@
+import ServerStateExample from './ServerStateExample';
+
+<CH.Scrollycoding>
+
+    <CH.Preview frameless>
+        <ServerStateExample />
+    </CH.Preview>
+
+    ```tsx ServerStateExample.tsx
```

Comment:
I think you want to use the code from the ServerStateExample.tsx file or?

```suggestion
    ```tsx ServerStateExample.tsx
     // from ./ServerStateExample.tsx
     ```
```